src/www/ui/api/documentation/openapi.yaml

```diff
@@ -3273,6 +3329,49 @@ components:
           description: Date from which to remove older log files from repository.
           nullable: true
           example: "2022-07-16"
+    ScheduleBulkScan:
+      type: object
+      properties:
+        bulkActions:
+          type: array
+          items:
+            type: object
+            properties:
+              licenseId:
+                type: integer
+                example: 1
+              reportinfo:
+                type: string
+                example: "License text"
+              acknowledgement:
+                type: string
+                example: "Acknowledgement text"
+              comment:
+                type: string
+                example: "Comment text"
+        refText:
+          type: string
+          example: "reference text"
+        bulkScope:
+          type: string
+          enum:
+            - local
+            - global
```

Comment:
Better names are ~local~ `folder` and ~global~ `upload`.

Reply:
Let me know if the changes look great now, @GMishx